app/assets/stylesheets/components/_spinner-button.scss

```diff
@@ -1,7 +1,86 @@
-.spinner-button--spinner-active {
-  @include u-text('center');
+@keyframes spinner-button-dot-one {
+  0% { transform: scale(0); }
+  25% { transform: scale(1); }
+  50% { transform: scale(0); }
+}
+
+@keyframes spinner-button-dot-two {
+  0% { transform: scale(0); }
+  20% { transform: scale(0); }
+  45% { transform: scale(1); }
+  70% { transform: scale(0); }
+}
+
+@keyframes spinner-button-dot-three {
+  0% { transform: scale(0); }
+  40% { transform: scale(0); }
+  65% { transform: scale(1); }
+  90% { transform: scale(0); }
+}
 
+.spinner-button--spinner-active {
   .spinner-button__content {
-    @include u-display('none');
+    display: inline-block;
```

Comment:
Per minor display difference in animation when clicking button, probably need to apply `display` styling consistently regardless of whether spinner is active or not.

Reply:
> Per minor display difference in animation when clicking button, probably need to apply `display` styling consistently regardless of whether spinner is active or not.

This was surprisingly difficult to fix. The issue is that the verify button tries to use flex grid columns to expand the button to occupy the full-width on mobile, and shrink to a smaller size on desktop. Neither of these are what we expect to be a "default" behavior, so reconciling it with other usage was not straight-forward.

The changes in dd065fc are what I landed at as a solution.